lua/lspconfig/jdtls.lua

```diff
@@ -1,33 +1,35 @@
+local os = require 'os'
```

Comment:
this is unecessary

Reply:
Thank you for pointing out, didn't know that!